scripts/php/baphpel.php

```diff
@@ -20,7 +20,7 @@
 ********************************************************************/
 namespace baphpel;
 
-include_once 'openbabel.php';
+include_once '/usr/local/lib/openbabel.php';
```

Comment:
Is this change intentional as part of this commit?

Reply:
No, not sure where that came from